tools/UnityPaths/UnityPaths.cs

```diff
@@ -53,6 +53,11 @@ public static string TryGetUnityPath()
                 return hubPath;
             }
 
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && Directory.Exists(Path.Combine(DefaultUnityPath, "Unity.App")))
```

Comment:
I just installed Unity with the Hub on my Mac and the path it is installed to is /Applications/Unity/Hub/Editor/<version>/Unity.app
Do we want to cover that scenario?

Reply:
that is already covered afaik

Reply:
Okay, reading the code, I guess that should be covered by the code above. If that is the case, ignore my comment

Reply:
Aye, if it's the right version it should